### PR TITLE
session_test: move rowlock and autocommit test.

### PR DESCRIPTION
**Pull request by hanfei1991**

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: session_test.go [hunk 1]
```diff
@@ -379,51 +338,6 @@ func (s *testSessionSuite) TestInTrans(c *C) {
...
-	// se1 will retry and the final value is 21
-	c.Assert(err, IsNil)
-	// Check the result is correct
-	se3 := newSession(c, s.store, dbName)
-	r := mustExecSQL(c, se3, "select c2 from t where c1=11")
-	rows, err := GetRows(r)
-	matches(c, rows, [][]interface{}{{21}})
-
-	mustExecSQL(c, se1, "begin")
-	mustExecSQL(c, se1, "update t set c2=21 where c1=11")
```
Comment (zimulala) [inline session_test.go]: Why remove this test?

File: new_session_test.go [hunk 2]
```diff
@@ -149,3 +149,64 @@ func (s *testSessionSuite) TestAffectedRows(c *C) {
...
+	tk1.MustExec("update t set c2=21 where c1=11")
+
+	tk2.MustExec("begin")
+	tk2.MustExec("update t set c2=211 where c1=11")
+	tk2.MustExec("commit")
+
+	// se1 will retry and the final value is 21
+	tk1.MustExec("commit")
+	// Check the result is correct
+	tk.MustQuery("select c2 from t where c1=11").Check(testkit.Rows("211"))
```
Comment (zimulala) [inline new_session_test.go]: Here the result is `211`, the old test is `21`.

File: session_test.go (hunk 1 above)
Comment (zimulala) [reply session_test.go]: PTAL @shenli

**Comment by shenli** [comment]
LGTM
/run-all-tests

**Comment by zimulala** [comment]
LGTM

**Pull request merged**